reactor-core/src/main/java/reactor/core/publisher/FluxReplay.java

```diff
@@ -1161,6 +1161,20 @@ public void subscribe(CoreSubscriber<? super T> actual) {
 		}
 	}
 
+	public static void main(String[] args) throws Exception {
+		ConnectableFlux<Integer> f = Flux.just(1, 2, 3).replay(0);
+
+		f.subscribe(System.out::println, System.out::println, () -> System.out.println(
+				"complete"));
+		f.connect();
+
+		f.subscribe(System.out::println, System.out::println, () -> System.out.println(
+				"complete"));
+
+		Thread.sleep(100);
+//		f.connect();
+	}
+
```

Comment:
I know this is still a draft but marking this so we don't forget to remove it 😄

Reply:
It's gone :)